pages/index.js

```diff
@@ -1,69 +1,42 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import { useState } from "react";
+import { nanoid } from "nanoid";
 
 export default function Home() {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
-
-        <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
 
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
+const [todos, setTodos] = useState ([
+  { description: "clean my unterhose",
+  done: false,
+  id: nanoid()},
+  { description: "clean my teeth",
+  done: false,
+  id: nanoid()}
+])
 
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
-          >
-            <h2>Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
-        </div>
-      </main>
-
-      <footer className={styles.footer}>
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <span className={styles.logo}>
-            <Image src="/vercel.svg" alt="Vercel Logo" width={72} height={16} />
-          </span>
-        </a>
-      </footer>
+  return (
+    <div>
+      <div>
+      <h1>TODO APP</h1>
+      <h3>TYPE YOUR TODOS HERE</h3>
+      <input type="text"></input>
+      <h3>NOCH ZU ERLEDIGEN:</h3>
+      </div>
+      <div>
+      {todos.map((todo)=>{
+        return(
+          <div>
+      <p>{todo.description}</p>
+      <button>Delete</button>
+      </div>
+        )
+      })}
+      </div>
+
+      
+
+
+      <h3>ERLEDIGT:</h3>
+      <p>Kochen</p>
+      <button>Delete</button>
```

Comment:
Du hast auch schon viel mehr Schritte eingebaut als in der ersten UserStory verlangt waren :) Der Delete-Button kommt z.B. eigentlich erst später, aber ist ja nicht schlimm. NUr fürs capstone-Projekt wäre es vielleicht besser, noch kleinteiliger vorzugehen.
Ansonsten, sieht alles gut aus für mich! Von mir aus kannst du mergen (ich hab diese schicke Abkürzung vergessen)